tests/ci/integration/run_bind9_integration.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash -exu
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  - SRC_ROOT(aws-lc)
+#    - SCRATCH_FOLDER
+#      - bind9
+#      - AWS_LC_BUILD_FOLDER
+#      - AWS_LC_INSTALL_FOLDER
+#      - BIND9_BUILD_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRATCH_FOLDER="${SRC_ROOT}/BIND9_BUILD_ROOT"
+BIND9_SRC_FOLDER="${SCRATCH_FOLDER}/bind9"
+BIND9_BUILD_FOLDER="${SCRATCH_FOLDER}/bind9-aws-lc"
+BIND9_PATCH_FOLDER=${SRC_ROOT}/"tests/ci/integration/bind9_patch"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${SCRATCH_FOLDER}/aws-lc-install"
+
+function bind9_build() {
+  autoreconf -fi  
+  PKG_CONFIG_PATH="${AWS_LC_INSTALL_FOLDER}/lib/pkgconfig" ./configure \
+      --with-openssl="${AWS_LC_INSTALL_FOLDER}" \
+      --enable-dnstap \
+      --enable-dnsrps \
+      --with-cmocka \
+      --with-libxml2 \
+      --enable-leak-detection
+  make -j ${NUM_CPU_THREADS} -k all
+}
+
+# TODO: Remove this when we make an upstream contribution.
+function bind9_patch() {
+  for patchfile in $(find -L "${BIND9_PATCH_FOLDER}" -type f -name '*.patch'); do
+    echo "Apply patch $patchfile..."
+    patch -p1 --quiet -i "$patchfile"
+  done
+}
+
+function bind9_run_tests() {
+  make -j ${NUM_CPU_THREADS} check
+}
+
+mkdir -p ${SCRATCH_FOLDER}
+rm -rf ${SCRATCH_FOLDER}/*
+cd ${SCRATCH_FOLDER}
+
+git clone https://gitlab.isc.org/isc-projects/bind9.git ${BIND9_SRC_FOLDER} --depth 1
+mkdir -p ${AWS_LC_BUILD_FOLDER} ${AWS_LC_INSTALL_FOLDER} ${BIND9_BUILD_FOLDER}
+ls
+
+aws_lc_build ${SRC_ROOT} ${AWS_LC_BUILD_FOLDER} ${AWS_LC_INSTALL_FOLDER} -DBUILD_TESTING=OFF -DBUILD_TOOL=OFF -DBUILD_SHARED_LIBS=1
```

Comment:
```suggestion
aws_lc_build ${SRC_ROOT} ${AWS_LC_BUILD_FOLDER} ${AWS_LC_INSTALL_FOLDER} -DBUILD_TESTING=OFF -DBUILD_TOOL=OFF -DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Reply:
I had consider using Release, but that would "work around" the assertion I removed. Does `RelWithDebInfo` trip on assertions?

Reply:
I did a test run and `RelWithDebInfo` doesn't trip over assertions. I'm fine either way, was just looking for 100% coverage of the changes I made here.

Reply:
Oh.... that's interesting, this makes me want to flip back more of our builds to just debug unless they actually need the speed like curl/mysql.

Reply:
Yeah I'm all for it. Our consumers should be using Release, so we wouldn't be accidentally breaking anyone with new assertions. But it'd be better to see if any integrations are using these code paths.